Gateway/Request/GooglePayAuthorizationDataBuilder.php

```diff
@@ -67,6 +76,10 @@ public function build(array $buildSubject)
             $this->adyenLogger->addAdyenDebug("PaymentToken is empty");
         }
 
+        $requestBody['returnUrl'] = $this->storeManager->getStore()->getBaseUrl(
```

Comment:
Should we use the PWA origin conditionally as we did here? https://github.com/Adyen/adyen-magento2/blob/7.0.0-rc.1/Block/Transparent/Redirect.php#L74-L80

Reply:
let's validate it with the test merchants because I am not sure if it's needed here as well